src/styles/blocks/header.scss

```diff
@@ -0,0 +1,86 @@
+@import '../utils/variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  padding: 0 50px;
+
+  border-bottom: 1px solid $border-color;
+
+  &-logo {
+    .logo {
+      height: 40px;
+      width: 40px;
+    }
+  }
+}
+
+// nav>list>list-item>list-item__link
+.nav {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
The `.nav` block seems to be a standalone component, but it's styling is included in the `header.scss` file. It would be better to move the `.nav` related styles to a separate SCSS file named `nav.scss` to follow the separation of concerns and single responsibility principles.